### PR TITLE
headerbar: increase headerbar backdrop contrast

### LINKED ISSUE #1740: Titlebar/headerbar of inactive windows
In Yaru theme (not light, not dark) the titlebars/headerbars of inactive windows are the same color (or almost) of active windows; It should be more like in Yaru-dark that is significantly different.

### PR DESCRIPTION
**Pull request by clobrano**
closes #1740

Before
image

After
image

### CODE REVIEW

**Comment by madsrh** [comment]
This increases headerbar backdrop contrast for sure ✔️ 👍 but that means that the black button borders are even more pronounced now. Can we lighten up (or remove) the borders in backdrop or is that a separate PR/issue?

IMHO removing the border would be the "prettiest" solution but that might cause some XU problems IDK.

Before:
image

After:
image

**Comment by clobrano** [comment]
I agree that the borders are too dark in backdrop, I thought of doing a different PR for that however

File: gtk/src/default/gtk-3.20/_headerbar.scss [hunk 1]
```diff
@@ -14,7 +14,7 @@ headerbar,
   }
 
   &:backdrop {
-    border-color: mix(darken($inkstone, 11%), darken($inkstone, 1%), 80%);
+    border-color: $inkstone;
```
Comment (Feichtmeier) [inline gtk/src/default/gtk-3.20/_headerbar.scss]: How about making use of the https://github.com/ubuntu/yaru/blob/684cf87538ac41ea5ac45da309bf0f130daad7b2/gtk/src/default/gtk-3.20/_colors.scss#L85 variable instead? So if you change the headerbar color the border adapts to it
For the backdrop you could add a backdrop variable

Comment (clobrano) [reply gtk/src/default/gtk-3.20/_headerbar.scss]: The goal was to adapt the BG color only and I removed the overly complicated expression for border color with inkstone, which is basically the se, but I agree

**Comment by clobrano** [comment]
> I agree that the borders are too dark in backdrop, I thought of doing a different PR for that however

alright, since I have to change some border colors anyway, I changed my mind.

@madsrh, what do you think about this lighter borders for buttons in backdrop?

right now
image

using hb color for borders
image

2% lighter hb color (it's barely visible for me)
image

**Comment by madsrh** [comment]
Much improved, thank you - this was a torn in my eye 😄 🎉 So I'm happy with both.

**Comment by clobrano** [comment]
very well, merging it then

**Pull request merged**